src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -316,7 +359,10 @@ class WorkspaceMembersPage extends React.Component {
                             title={this.props.translate('workspace.common.members')}
                             subtitle={policyName}
                             onCloseButtonPress={() => Navigation.dismissModal()}
-                            onBackButtonPress={() => Navigation.navigate(ROUTES.getWorkspaceInitialRoute(policyID))}
+                            onBackButtonPress={() => {
+                                this.clearMembersAndSearchValue();
```

Comment:
Why do we need to do this?

Reply:
@marcaaron its fixing issue state reset when page navigation to back or invite page then come back to Members list page

Reply:
Yeah I'm not sure we need that. Let's remove it for now? And then also clean up the method called `clearMembersAndSearchValue()`. It's not really needed as a method if we only use it once and we can simplify this code. Thanks!

Reply:
Ok @marcaaron i will remove it for now